gobcore/logging/logger.py

```diff
@@ -47,6 +52,20 @@ def emit(self, record):
         RequestsHandler.LOG_PUBLISHER.publish(record.levelname, log_msg)
 
 
+class ExtendedLogger(logging.Logger):
+    def data_info(self, msg, *args, **kwargs):
```

Comment:
Top!

I propose to have all constants like DATAINFO etc and the addLevels code in this class as class variables and methods. This keeps the code more consistent and maintainable

Reply:
Also add some comments in this class why it is created and what its responsabilites are

Reply:
Done